core/agent.py

```
@@ -207,26 +254,73 @@ def chat(self, max_iterations: Optional[int] = None):
+                # 记录产物（仅在任务模式下）
+                if self.spec and isinstance(result, dict) and "file_path" in result:
+                    self._record_artifact(result["file_path"])
+
+                # 将结果添加到对话
+                self.conversation.append(
+                    {
+                        "role": "user",
+                        "content": f"tool_result({json.dumps(result, ensure_ascii=False)})",
+                    }
+                )
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)